http_server/routes/routes.go

```diff
@@ -31,12 +32,12 @@ func (lw loggerWriter) Write(p []byte) (int, error) {
 	splitted := strings.Split(log, "| ")
 	statusCode := strings.Trim(splitted[1], " ")
 	if statusCode != "200" && statusCode != "204" {
-		logger.Error(log)
+		fmt.Errorf(log)
 	}
 	return len(p), nil
 }
 
-func InitializeHttpRoutes() *gin.Engine {
+func InitializeHttpRoutes(handlers *handlers.Handlers) *gin.Engine {
 	gin.DefaultWriter = loggerWriter{}
```

Comment:
we can remove this line + the loggerWriter + write ffunction, we don't need this anymore